packages/console/src/ConsoleMenu.tsx

```diff
@@ -187,6 +187,9 @@ class ConsoleMenu extends PureComponent<ConsoleMenuProps, ConsoleMenuState> {
           kind="ghost"
           className="btn-link-icon"
           disabled={tableActions.length === 0}
+          onClick={() => {
+            // no-op: click is handled in `AppControlsMenu` (which uses a `DropdownMenu`)
```

Comment:
Comment doesn't make sense - this isn't using `AppControlsMenu` here. Just say click is handled in `DropdownMenu`:

```suggestion
            // no-op: click is handled in `DropdownMenu`
```